pkg/api/search.go

```diff
@@ -28,6 +28,7 @@ func (hs *HTTPServer) Search(c *contextmodel.ReqContext) response.Response {
 	page := c.QueryInt64("page")
 	dashboardType := c.Query("type")
 	sort := c.Query("sort")
+	panelTitle := c.Query("panelTitle")
```

Comment:
nit; I wonder if we better allow users to pass multiple panel title to the request

Reply:
I'll check if the [Product DNA](https://docs.google.com/document/d/1u6NdbqGmz7beoByKUoxSu_2HQLF4iz1JAKSph5iTOHo/edit) addresses this point. Otherwise it would be worth adding as a question. For now I would assume we want to reproduce search V2 functionality. That's something else to figure out--what search V2 does in terms of the option of passing multiple panel titles.